components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/bindings/handlers/TokenBindingExpiryEventHandler.java

```diff
@@ -100,6 +113,124 @@ public void handleEvent(Event event) throws IdentityEventException {
         }
     }
 
+    /**
+     * This method will get the application information from session context and revoke access tokens of the
+     * applications bound to that session. This method can be used when token binding information id not found in the
```

Comment:
..token binding information **id** not found in the... -> ...token binding information is not found in the...